scripts/jest/setup.js

```diff
@@ -19,6 +19,8 @@
 import 'jest-axe/extend-expect';
 import '@testing-library/jest-dom';
 import 'raf/polyfill';
+import { TextEncoder } from 'util'
```

Comment:
This mock is required as part of the upgrade as in a newer version of jsdom this is not exposed/available to the environment so causes tests to fail due to underlying global not being available